src/main/js/controls/Timeline/TimelineContent.js

```diff
@@ -203,6 +203,38 @@ class TimelineContent extends GraphContent {
         return this;
     }
 
+    /**
+     * @inheritdoc
+     */
+    reflow(graph, graphData) {
+        this.config.values = graphData.values; // Update the old Scatter config values with the new provided values
+        this.dataTarget = processDataPoints(graph.config, this.config); // Update the dataTarget
```

Comment:
What is the difference between the parameters `graph.config` and `this.config`?

Reply:
`graph.config` holds the config for the timeline construct. And `this.config` holds the config for graph content.